src/darkmode.js

```diff
@@ -1,17 +1,15 @@
 export default class Darkmode {
   constructor(options) {
-    const bottom = options && options.bottom ? options.bottom : '32px';
-    const right = options && options.right ? options.right : '32px';
-    const left = options && options.left ? options.left : 'unset';
-    const time = options && options.time ? options.time : '0.3s';
-    const mixColor = options && options.mixColor ? options.mixColor : '#fff';
-    const backgroundColor = options && options.backgroundColor ? options.backgroundColor : '#fff';
-    const buttonColorDark = options && options.buttonColorDark ? options.buttonColorDark : '#100f2c';
-    const buttonColorLight = options && options.buttonColorLight ? options.buttonColorLight : '#fff';
-    const label = options && options.label ? options.label : '';
-    /* eslint-disable */
-    const saveInCookies = options && options.saveInCookies === false ? false : true;
-    /* eslint-enable */
+    const bottom = options && options.bottom || '32px';
+    const right = options && options.right || '32px';
+    const left = options && options.left || 'unset';
+    const time = options && options.time || '0.3s';
+    const mixColor = options && options.mixColor || '#fff';
+    const backgroundColor = options && options.backgroundColor || '#fff';
+    const buttonColorDark = options && options.buttonColorDark || '#100f2c';
+    const buttonColorLight = options && options.buttonColorLight || '#fff';
+    const label = options && options.label || '';
+    const saveInCookies = options && options.saveInCookies;
```

Comment:
```suggestion
    const saveInCookies = options && !!options.saveInCookies;
```
to ensure it's a boolean

Reply:
In that case, I'd prefer `options && Boolean(options.saveInCookies)`